docs/source/cpp/compute.rst

```diff
@@ -98,6 +98,8 @@ exact semantics of the function::
    min_value = min_max.scalar_as<arrow::StructScalar>().value[0];
    max_value = min_max.scalar_as<arrow::StructScalar>().value[1];
 
+However, stateful compute functions like `Grouped Aggregations <#grouped-aggregations-group-by>`_ are not invocable via ``CallFunction``
+
```

Comment:
A few things:

- I think it's only grouped aggregations that cannot be invoked this way, so can we just state that directly?
- We should use the proper reST syntax for linking to a section.
- Can we break the line to be roughly consistent with the rest of the file?
- Just a small typo: there's no period at the end of the sentence.

Reply:
@lidavidm made the changes